reports/pin/pin.qmd

```diff
@@ -76,81 +279,329 @@ assessment_pin %>%
+    "Final FMV Bldg No Prorate", "Prior Far Total"
+  ))
 ```
 
-## Sales
 
-```{r _pin_topline_characteristics, results= 'asis'}
-training_data %>%
+
+```{r}
+assessment_pin %>%
+  mutate(township = ccao::town_convert(meta_township_code)) %>%
   filter(meta_pin == target_pin) %>%
-  arrange(desc(meta_sale_date)) %>%
+  arrange(meta_pin) %>%
   select(
-    "Sale Date" = meta_sale_date,
-    "Sale Price" = meta_sale_price,
-    "Document Number" = meta_sale_document_num,
-    "Outlier Type" = sv_outlier_type
-  ) %>%
-  mutate(`Sale Price` = as.numeric(`Sale Price`)) %>%
-  mutate(`Sale Price` = scales::dollar(`Sale Price`)) %>%
-  kable(align = "lrll") %>%
-  kable_styling(
-    "striped",
-    position = "left",
-    fixed_thead = TRUE
-  )
+    "Township" = township,
+    "Prior Far Total" = prior_far_tot,
+    "Prior Near Total" = prior_near_tot,
+    "Initial Pred. FMV" = pred_pin_initial_fmv,
+    "Final Pred. FMV" = pred_pin_final_fmv,
+    "Final Pred. FMV Round" = pred_pin_final_fmv_round,
+    "Final FMV Land" = pred_pin_final_fmv_land,
+    "Final FMV Round No Prorate" = pred_pin_final_fmv_round_no_prorate,
+    "Final FMV Bldg" = pred_pin_final_fmv_bldg,
+    "Final FMV Bldg No Prorate" = pred_pin_final_fmv_bldg_no_prorate,
+    "Bldg Effective" = pred_pin_bldg_rate_effective,
+    "Land Pct. Total" = pred_pin_land_pct_total,
+    "Land Rate Effective" = pred_pin_land_rate_effective,
+    sale_recent_1_date,
+    sale_recent_2_date,
+    sale_recent_1_price,
+    sale_recent_1_document_num,
+    sale_recent_2_price,
+    sale_recent_2_document_num,
+    prior_near_yoy_change_pct
+  ) %>%
+  mutate(
+    across(
+      c(`Prior Far Total`:`Bldg Effective`, `Land Rate Effective`),
+      ~ scales::dollar(.x)
+    ),
+    across(
+      `Land Pct. Total`,
+      ~ scales::percent(.x, accuracy = 0.01)
+    )
+  ) %>%
+  mutate(
+    "Sale Date 1" = as.character(sale_recent_1_date),
+    sale_recent_1_price = as.numeric(sale_recent_1_price),
+    "Sale Price 1" = scales::dollar(sale_recent_1_price),
+    "Sale Doc. 1" = sale_recent_1_document_num,
+    "Sale Date 2" = as.character(sale_recent_2_date),
+    "Sale Price 2" = scales::dollar(sale_recent_2_price),
+    "Sale Doc. 2" = sale_recent_2_document_num,
+    "YoY % Value Change" = scales::percent(prior_near_yoy_change_pct)
+  ) %>%
+  help_clean_tables() %>%
+  replace(is.na(.), "-") %>%
+  gt() %>%
+  tab_options(
+    table.width = pct(100)
+  ) %>%
+  cols_hide(columns = c(
+    sale_recent_1_date, sale_recent_2_date,
+    sale_recent_1_price, sale_recent_2_price,
+    sale_recent_1_document_num, sale_recent_2_document_num,
+    prior_near_yoy_change_pct, "Final FMV Land",
+    "Final FMV Round No Prorate", "Final FMV Bldg",
+    "Final FMV Bldg No Prorate", "Prior Far Total"
+  ))
 ```
 
-## Characteristics
 
-```{r _pin_characteristics}
-chars_data %>%
-  filter(meta_pin == target_pin) %>%
+
+
+## Multi-Card & Multi-PIN Aggregation 
```

Comment:
**issue (blocking):** This table is still pretty confusing. I really think we need to add a visual example of the aggregation. Do you have the notes that I created before? Can you attach them to a comment responding below?